src/views/product-downloads-view/index.tsx

```diff
@@ -37,7 +37,7 @@ import {
 	SidecarMarketingCard,
 } from './components'
 import s from './product-downloads-view.module.css'
-
+import { useRouter } from 'next/router'
```

Comment:
⛏️ style nit: can we reorder this import to the top? We try to organize imports in groups, from external (top) to local (bottom)